tests/integration/test_integration_all_rust.py

```diff
@@ -584,7 +579,7 @@ def teardown_module():
     kill_process(EP_SERVER)
 
 
-class TestRustWebPush(unittest.TestCase):
+class TestRustWebPush:
     """Test class for Rust Web Push."""
```

Comment:
We can get rid of this class and use pytest fixtures but we can do that later on.

Reply:
@jrbenny35  absolutely! That's the next ticket once this is done and am very excited to have fixtures over the class. Thanks for chiming in!